taxi/urls.py

```diff
@@ -40,9 +44,23 @@
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
+    path(
+        "car/toggle_driver/", ToggleDriverView.as_view(), name="toggle-driver"
```

Comment:
Consider renaming the URL pattern to 'car-toggle-driver' for consistency with other car-related URL patterns.